measurement_kit.go

```diff
@@ -3,15 +3,18 @@ package mk
 /*
 #include <measurement_kit/ffi.h>
 */
-// #cgo CFLAGS: -I${SRCDIR}/include
-// #cgo LDFLAGS: -lc++
+// #cgo darwin CFLAGS: -I${SRCDIR}/libs/include
+// #cgo darwin LDFLAGS: -lc++
 // #cgo darwin,amd64 LDFLAGS: ${SRCDIR}/libs/libevent-darwin-amd64.a
 // #cgo darwin,amd64 LDFLAGS: ${SRCDIR}/libs/libevent_openssl-darwin-amd64.a
 // #cgo darwin,amd64 LDFLAGS: ${SRCDIR}/libs/libevent_pthreads-darwin-amd64.a
 // #cgo darwin,amd64 LDFLAGS: ${SRCDIR}/libs/libcrypto-darwin-amd64.a
 // #cgo darwin,amd64 LDFLAGS: ${SRCDIR}/libs/libssl-darwin-amd64.a
 // #cgo darwin,amd64 LDFLAGS: ${SRCDIR}/libs/libGeoIP-darwin-amd64.a
 // #cgo darwin,amd64 LDFLAGS: ${SRCDIR}/libs/libmeasurement_kit-darwin-amd64.a
+// #cgo windows CFLAGS: -I${SRCDIR}/libs/include
```

Comment:
You don't have to duplicate this since it's the same for macOS, linux and windows.

Reply:
Yeah, I should have unified flags in 84e6051